### ISSUE TITLE
Log events such as ++ and new comments

### ISSUE THREAD

**Issue by allanx2000** [devRant/devRant#429]
Then you could generate a time graph for each post/user using ELK. Just something I thought of... ¯\_(ツ)_/¯

**Comment by 7twin** [comment]
@Skayo made something like that for Devrant++ Members: https://devrant-stats.github.io/

**Comment by Skayo** [comment]
@7twin Thanks for mentioning my project! Btw isn't only for devRant++ anymore. You can send a request to get added manually.

**Comment by DFoxinator** [comment]
@allanx2000 Where would you want to see the events get added? Some more info on this would be helpful.

@Skayo stats app is looking pretty cool, but can you please remove the devRant++ stats? They are wildly inaccurate (showing almost double the members/$), so they paint a very incorrect picture.

**Comment by Skayo** [comment]
Yeah sorry it currently counts all users in my DB but since not all of them are supporters it's inaccurate. Will fix asap. Eventually remove.
